core-primitives/enclave-api/src/remote_attestation.rs

```diff
@@ -110,6 +117,26 @@ pub trait TlsRemoteAttestation {
 	) -> EnclaveResult<()>;
 }
 
+impl Enclave {
+	unsafe fn get_collateral(
+		&self,
+		fmspc: [u8; 6],
```

Comment:
How about making the `[u8; 6]` into its own type alias to reduce code duplication?

Reply:
Good point. I added the type alias, but there is actually a type alias for that in the pallets but this is not yet merged. For now the type is duplicated but I added a comment to remove this in an existing cleanup issue https://github.com/integritee-network/pallets/issues/119